src/onyx/messaging/aeron.clj

```diff
@@ -17,6 +17,38 @@
              [java.util.function Consumer]
              [java.util.concurrent TimeUnit]))
 
+(def max-port 50000)
+
+(defrecord AeronPeerGroup [opts]
+  component/Lifecycle
+  (start [component]
+    (taoensso.timbre/info "Starting Aeron")
+    (let [media-driver (MediaDriver/launch)]
+      (assoc component :media-driver media-driver)))
+
+  (stop [{:keys [media-driver] :as component}]
+    (.close media-driver)
+    (assoc component :media-driver nil)))
+
+(defn aeron-peer-group [opts]
+  (map->AeronPeerGroup {:opts opts}))
+
+(defmethod extensions/assign-site-resources :aeron
```

Comment:
@MichaelDrogalis dispatch via keyword here means we don't have to actually setup a messaging component when playing back the log (which would also mean we'd need the config). I could see forgetting about it and trying to dispatch on type though.